client/components/progress-bar/index.jsx

```diff
@@ -28,14 +29,25 @@ export default class ProgressBar extends PureComponent {
 		compact: PropTypes.bool,
 		className: PropTypes.string,
 		isPulsing: PropTypes.bool,
+		canGoBackwards: PropTypes.bool,
```

Comment:
I thought this would be useful 😀 

https://github.com/Automattic/wp-calypso/pull/19015#issuecomment-338918324